tags.go

```diff
@@ -54,3 +54,14 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// TagExists returns true if tag exists in tags
+func TagExists(tags Tags, tagValue string) bool {
+	for _, tag := range tags.Split() {
+		if strings.ToLower(tag) == strings.ToLower(tagValue) {
```

Comment:
I would create a new variable for `strings.ToLower(tagValue)` as we're running this function on a constant 🙂